_posts/pl/2021-02-01-odpowiedzi-na-pytania-cd-tech-5.md

```diff
@@ -0,0 +1,115 @@
+---
+layout:    post
+title:     "Consdata Tech #5 - odpowiedzi na pytania!"
+date:      2021-02-01 08:00:00 +0100
+published: true
+lang:      pl
+author:    mkociszewski
+image:     /assets/img/posts/2021-02-01-odpowiedzi-na-pytania-cd-tech-5/lightboard.png
+tags:
+- ddd
+- axon
+- consdata tech
+- cdtech#5
+---
+
+Zgodnie z obietnicą, przyszedł czas odpowiedzieć na pytania, które pozostały bez odpowiedzi podczas naszego minionego eventu - Consdata Tech.
```

Comment:
IMO fajnie by było dać linka albo osadzić filmik YT z wydarzenia. Jeśli byś osadzał odtwarzacz YT to należy to zrobić includując plik youtube.html (w ten sposób bedziesz mieć pewność, że odtwarzacz będzie dobrze ostylowany i będzie ładnie wyglądać we wszystkich rozdzielczościach) np:
```
{% include youtube.html movie="2_jwbTviNoI" %}
```
Przykład masz w 2016-12-22-hystrix-praktyczne-uzycie-circuit-breakera.md

Reply:
Dzięki! Brzmi jak plan